CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ For larger features, we'd appreciate it if you open a [new issue](https://github
 To hack on Pulumi, you'll need to get a development environment set up. You'll want to install the following on your machine:
 
 - Go 1.16
-- NodeJS 10.X.X or later
+- NodeJS 11.X.X or later
```

Comment:
It's really 12 or later. 12 is the current Maintenance LTS version. https://nodejs.org/en/about/releases/